WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -133,15 +133,24 @@ public async Task<bool> StartCoinJoinAsync(IEnumerable<SmartCoin> coinCandidates
 
 	/// <summary>Attempt to participate in a specified round.</summary>
 	/// <param name="roundState">Defines the round parameter and state information to use.</param>
-	/// <returns>Whether or not the round resulted in a successful transaction.</returns>
+	/// <returns><c>True</c>: client should end the current CoinJoin progress. <c>False</c>: continue with the blame rounds.</returns>
```

Comment:
I am not sure, shouldn't this be the opposite?

Reply:
Not the scope of this PR to change this. I would leave these kinds of changes at the end when the CoinJoinClient is stable enough.

Reply:
Even then, it shouldn't be the opposite. When you signal error with a boolean return, the question is: `IsSuccessful`. By specifying what the consumer of the function should do, it may confused Yahia to think the question is: `IsBlameround`.